driver/kubedriver.go

```diff
@@ -161,6 +162,12 @@ func (driver *KubeDriver) getSpecSecretName() string {
 	return fmt.Sprintf("%v-spec", driver.Group)
 }
 
+// getMetricsSpecSecretName yields the name of the experiment metrics spec
+// secret
+func (driver *KubeDriver) getMetricsSpecSecretName(provider string) string {
```

Comment:
I don't think we want a separate secret for metrics. 

We will use the same secret used for storing the experiment spec. This will give us two secrets always, one for experiment/metric specs, another for result.